charms/worker/k8s/src/charm.py

```diff
@@ -322,8 +326,9 @@ def _bootstrap_k8s_snap(self):
         bootstrap_config = BootstrapConfig.construct()
         self._configure_datastore(bootstrap_config)
         bootstrap_config.cluster_config = self._assemble_cluster_config()
-        bootstrap_config.service_cidr = str(self.config["service-cidr"])
-        bootstrap_config.control_plane_taints = str(self.config["register-with-taints"]).split()
+        bootstrap_config.service_cidr = str(self.config["bootstrap-service-cidr"])
+        bootstrap_config.pod_cidr = str(self.config["bootstrap-pod-cidr"])
+        bootstrap_config.control_plane_taints = str(self.config["bootstrap-node-taints"]).split()
```

Comment:
Do we need to rename this attr? Are these taints only related to control plane nodes?

Reply:
on bootstrap, yes they're only related to control-planes

https://github.com/canonical/k8s-snap-api/blob/main/api/v1/type_bootstrap_config.go#L14

Reply:
Correct, thanks a lot!